definitions/divorce/json/CaseEventToFields/CaseEventToFields-nonprod.json

```diff
@@ -205,60 +205,78 @@
     "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "EnterAOSOffline_CoRespondent",
-    "CaseFieldID": "D8ReasonForDivorceAdultery3rdAddress",
-    "PageFieldDisplayOrder": 10,
-    "DisplayContext": "OPTIONAL",
+    "CaseFieldID": "CoRespSolAOSSOT",
+    "PageFieldDisplayOrder": 14,
+    "DisplayContext": "MANDATORY",
     "PageID": "aosdataentrycoresp1",
     "PageLabel": "Enter scanned form case data",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
+    "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Yes\"",
     "ShowSummaryChangeOption": "Yes"
   },
   {
     "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "EnterAOSOffline_CoRespondent",
-    "CaseFieldID": "CoRespEmailAddress",
-    "PageFieldDisplayOrder": 11,
-    "DisplayContext": "OPTIONAL",
+    "CaseFieldID": "CoRespondentSolicitorName",
+    "PageFieldDisplayOrder": 15,
+    "DisplayContext": "MANDATORY",
     "PageID": "aosdataentrycoresp1",
     "PageLabel": "Enter scanned form case data",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
+    "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Yes\"",
     "ShowSummaryChangeOption": "Yes"
   },
   {
     "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "EnterAOSOffline_CoRespondent",
-    "CaseFieldID": "CoRespPhoneNumber",
-    "PageFieldDisplayOrder": 12,
+    "CaseFieldID": "CoRespondentSolicitorCompany",
+    "PageFieldDisplayOrder": 16,
+    "DisplayContext": "MANDATORY",
```

Comment:
This one needs to be optional I think from what the story says :)

Reply:
Thanks, missed that.